<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Thanh_ly</class>
 <widget class="QDialog" name="Thanh_ly">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget_thanhly">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>491</width>
     <height>231</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Thanh lý</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Mã sách</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Vị trí</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Trạng thái</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>541</width>
     <height>90</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>13</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>ISBN :</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_ISBN"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Tên sách :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_tensach"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Số trang :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="lineEdit_sotrang"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>158</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>13</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Tác giả :</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_tacgia"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Năm suất bản :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="lineEdit_namsb"/>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>138</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>18</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Thể loại :</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_theloai"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_chontatca">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>370</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #28C3E5;
    border: none;
    border-radius: 12px; /* Giảm border-radius để nút nhỏ hơn */
    padding: 8px 10px; /* Giảm padding */
    font-family: &quot;Roboto Mono&quot;, sans-serif;
    font-size: 14px; /* Giảm cỡ chữ */
    font-weight: bold; /* Giữ độ dày chữ */
    letter-spacing: 0.8px;
    color: #FFFFFF;
    text-align: center;
    width: 100%;
}

QPushButton:hover {
    background-color: #1CB0F6;
}

QPushButton:pressed {
    background-color: #1899D6;
    border-width: 4px 0 0;
}

QPushButton:disabled {
    background-color: #D6D6D6;
    color: #7F7F7F;
}

QPushButton:focus {
    outline: none;
}
</string>
   </property>
   <property name="text">
    <string>Chọn tất cả</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_bochon">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>370</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #28C3E5;
    border: none;
    border-radius: 12px; /* Giảm border-radius để nút nhỏ hơn */
    padding: 8px 10px; /* Giảm padding */
    font-family: &quot;Roboto Mono&quot;, sans-serif;
    font-size: 14px; /* Giảm cỡ chữ */
    font-weight: bold; /* Giữ độ dày chữ */
    letter-spacing: 0.8px;
    color: #FFFFFF;
    text-align: center;
    width: 100%;
}

QPushButton:hover {
    background-color: #1CB0F6;
}

QPushButton:pressed {
    background-color: #1899D6;
    border-width: 4px 0 0;
}

QPushButton:disabled {
    background-color: #D6D6D6;
    color: #7F7F7F;
}

QPushButton:focus {
    outline: none;
}
</string>
   </property>
   <property name="text">
    <string>Bỏ chọn</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>400</y>
     <width>168</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QPushButton" name="pushButton_thanhly">
      <property name="text">
       <string>Thanh lý</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_cancel">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
